WcaOnRails/app/models/user.rb

```diff
@@ -323,12 +327,16 @@ def can_admin_results?
     admin? || board_member? || results_team?
   end
 
-  # Team leaders should be able to edit their team.
-  # See https://github.com/cubing/worldcubeassociation.org/issues/427
-  def can_edit_teams?
+  # Returns true if the user can perform every action for teams.
+  def can_manage_teams?
     admin? || board_member? || results_team?
   end
 
+  # Returns true if the user can edit the given team.
+  def can_edit_team?(team)
```

Comment:
I'm guessing what this means, but would a clearer name for this be "can change team members" or something?